src/ui.cpp

```diff
@@ -12,6 +12,7 @@
 #include "options.h"
 #include "game.h"
 #include "player.h"
+#include "cursesdef.h"
```

Comment:
Why do you need to add an include, when you only *remove* code. The header is btw. already included in ui.h